pkg/api/k8s_versions.go

```diff
@@ -832,8 +832,8 @@ func getK8sVersionComponents(version, kubernetesImageBaseType string, overrides
 			common.ControllerManagerComponentName:             getDefaultImage(common.ControllerManagerComponentName, kubernetesImageBaseType) + ":v" + version,
 			common.KubeProxyAddonName:                         getDefaultImage(common.KubeProxyAddonName, kubernetesImageBaseType) + ":v" + version,
 			common.SchedulerComponentName:                     getDefaultImage(common.SchedulerComponentName, kubernetesImageBaseType) + ":v" + version,
-			common.CloudControllerManagerComponentName:        azureCloudControllerManagerImageReference,
-			common.CloudNodeManagerAddonName:                  azureCloudNodeManagerImageReference,
+			common.CloudControllerManagerComponentName:        "oss/kubernetes/azure-cloud-controller-manager:v0.6.0",
+			common.CloudNodeManagerAddonName:                  "oss/kubernetes/azure-cloud-node-manager:v0.6.0",
```

Comment:
Should we apply the same change to 1.18 where it uses azure-cloud-controller-manager v0.5.1 according to https://github.com/kubernetes-sigs/cloud-provider-azure#current-status?

Reply:
1.18 stuff is left as a historical artifact: you can't actually build a 1.18 using current aks-engine code.